hikari/errors.py

```diff
@@ -317,9 +317,6 @@ class RESTErrorCode(int, enums.Enum):
     MESSAGE_TOO_OLD = 50_034
     """A message provided was too old to bulk delete."""
 
-    TWO_FACTOR_AUTHENTICATION_REQUIRED = 60_003
-    """2FA is required to use this endpoint."""
```

Comment:
Why has this been removed?

https://github.com/discord/discord-api-docs/pull/1671 documents this, Discord just haven't bothered to merge it.

Please can we revert this.

Reply:
I thought that bots could not get this error, which is why I removed it. Will revert

Reply:
Sure, but the REST only client allows logging in via users over OAuth2 :)

Also I think bots may be able to get this in some edge cases possibly.